client/src/components/core/ms-input/MsPasswordInput.vue

```diff
@@ -38,9 +38,23 @@
 </template>
 
 <script setup lang="ts">
+import { HotkeyManager, HotkeyManagerKey, Modifiers, Platforms } from '@/services/hotkeyManager';
 import { IonCol, IonGrid, IonIcon, IonInput, IonItem, IonRow, IonText } from '@ionic/vue';
 import { eye, eyeOff } from 'ionicons/icons';
-import { ref } from 'vue';
+import { inject, onMounted, onUnmounted, ref } from 'vue';
+
+const hotkeyManager: HotkeyManager = inject(HotkeyManagerKey)!;
+const hotkeys = hotkeyManager.newHotkeys();
+
+onMounted(async () => {
+  await hotkeys.add('/', Modifiers.Ctrl, Platforms.Desktop | Platforms.Web, async () => {
```

Comment:
We should be careful about what components can register hotkeys. For example, what does this do in the change password page? Three components toggle the visibility at the same time, which is not what we want.

Registering the shortcut on the parent component can be a pain. In this case, I think checking that the component is in focus before toggling the visibility should be enough.